tests/conformance/conformance.go

```diff
@@ -36,17 +36,22 @@ func execute() error {
 	} else {
 		args = append(args, "--ginkgo.skip", "\\[Disruptive\\]")
 	}
-	if value, exists := os.LookupEnv("E2E_FOCUS"); exists {
+
+	if value, exists := os.LookupEnv("E2E_LABEL"); exists {
+		args = append(args, "--ginkgo.label-filter", value)
+	} else if value, exists := os.LookupEnv("E2E_FOCUS"); exists {
```

Comment:
do we want to handle a case both exist?
if so the focus part will be ignored here.